src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRuleList.tsx

```diff
@@ -110,7 +115,16 @@ function getListItems(
             onClick: () => onEditRule(item.id),
             overflowMenuItems: getMenuItems(item),
             textPrimary: item.displayName,
-            textSecondary: getSecondaryText(item, t)
+            textSecondary: getSecondaryText(item, t),
+            buttonProps: {
+                customStyles: {
```

Comment:
is it possible to move this styling to style class and assign it as style or classname here instead of setting the style explicitly here?

Reply:
if it requires to be a style object you can get advantage from subcomponents and instead of classname you can use it as style object and set here. I did it in IHighChartsWrapperStyles